src/mame/drivers/m72.cpp

```diff
@@ -1865,7 +1904,7 @@ void m72_state::m72(machine_config &config)
 {
 	m72_base(config);
 	/* Sample rate verified (Gallop : https://youtu.be/aozd0dbPzOw) */
-	m_soundcpu->set_periodic_int(FUNC(m72_state::fake_nmi), attotime::from_hz(32000000/8/512));
+	m_soundcpu->set_periodic_int(FUNC(m72_state::fake_nmi), attotime::from_ticks(8 * 512, 32000000));
```

Comment:
Something wrong with from_hz? I think hz is saner than ticks when reading code. There is no rounding error (maybe you thought there is and that's why you changed it?), see: if (attotime::from_ticks(8 * 512, 32000000) == attotime::from_hz(MASTER_CLOCK/8/512)) printf("same");

Reply:
Actually, you have more chance of a rounding error with from_ticks. For example in btoads, the from_ticks you added there a while ago:

attotime a = attotime::from_ticks(32768, XTAL(24'000'000)/4);
attotime b = attotime::from_hz(XTAL(24'000'000)/4/32768);

a == 0.005461333333311488
b == 0.005461333333333333